src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Filter.scss';
+
+const Filter = ({ filters }: { filters: string[] }) => {
```

Comment:
`Suggestion`

A common practice is to type it as `FilterProps`:

```tsx

type FilterProps = {
  filters: string[]; 
}

const Filter = ({ filters }: FilterProps) => {
  //...
}
```

You keep your function signature short, consistent throughout all your components,
and it's easier to grow/shrink when necessary.